session/session.go

```diff
@@ -2525,7 +2525,8 @@ func (s *session) checkPlacementPolicyBeforeCommit() error {
 	}
 	if txnScope != config.DefTxnScope {
```

Comment:
Change `config.DefTxnScope` to `oracle.GlobalTxnScope`. Variables in `config` is used for the default values for configurations.

Reply:
updated.